### ISSUE TITLE
ZipExtractor.exe is used by another process?

### ISSUE THREAD

**Issue by koolkiizz** [ravibpatel/AutoUpdater.NET#652]
On some machine, it throw the error:
![image](https://github.com/ravibpatel/AutoUpdater.NET/assets/20215671/038b4cbe-09cf-4cef-8401-a71549a6d1d8)

**Comment by drodz-dev** [comment]
Hi @koolkiizz, we encountered this same error and it appears to be caused by antivirus software. Some antivirus programs might interfere with ZipExtractor to function properly.

**Comment by ravibpatel** [comment]
Yes, It is possible that Antivirus software can cause this. As a possible workaround, you may try to use the installer file instead of the zip file as an update file.
